composer.json

```diff
@@ -25,20 +25,20 @@
         "php":                      "^7.0",
         "phpspec/prophecy":         "^1.5",
         "phpspec/php-diff":         "^1.0.0",
-        "sebastian/exporter":       "^1.0 || ^2.0",
-        "symfony/console":          "^2.7 || ^3.0",
-        "symfony/event-dispatcher": "^2.7 || ^3.0",
-        "symfony/process":          "^2.7 || ^3.0",
-        "symfony/finder":           "^2.7 || ^3.0",
-        "symfony/yaml":             "^2.7 || ^3.0",
+        "sebastian/exporter":       "^2.0",
+        "symfony/console":          "^3.2",
```

Comment:
Another comment by Ciaran yesterday is that PhpSpec 3.x will be around for 18 months or so yet. Whatever can be targeted at 3.x will be, it depends on whether PhpSpec 4 should support Symfony 2.7, given that the support lifetime of PhpSpec 3.x and Symfony 2.7 are similar, and how much time is left to support Symfony 2.7 once PhpSpec 4 is released.